examples/migraphx_docker/README.md

```diff
@@ -0,0 +1,9 @@
+# Building the MIGraphX Dockerfile
+
+You can build the provided dockerfile with the following command:
+
+    docker build -f Dockerfile.hip-clang -t migraphx .
+
+To run the docker image:
+
+    docker run --device='/dev/kfd' --device='/dev/dri' -v=`pwd`:/data -w /data --group-add video -it migraphx
```

Comment:
Are we going to download data for the data folder? If not, I think we should remove the -v='pwd':/data and -w /data from this command.